[2024-12-18T15:44:02.432+0000] {processor.py:186} INFO - Started process (PID=3627) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:44:02.435+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:44:02.440+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:44:02.438+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:44:02.447+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:44:02.444+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 3
    from datetimeimport datetime
                        ^^^^^^^^
SyntaxError: invalid syntax
[2024-12-18T15:44:02.448+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:44:02.490+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.070 seconds
[2024-12-18T15:45:14.057+0000] {processor.py:186} INFO - Started process (PID=3695) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:45:14.059+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:45:14.061+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:45:14.060+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:45:14.064+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:45:14.063+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 3
    from datetimeimport datetime
                        ^^^^^^^^
SyntaxError: invalid syntax
[2024-12-18T15:45:14.065+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:45:14.085+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.034 seconds
[2024-12-18T15:45:48.862+0000] {processor.py:186} INFO - Started process (PID=3765) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:45:48.863+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:45:48.865+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:45:48.865+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:45:48.869+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:45:48.868+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 3
    from datetimeimport datetime
                        ^^^^^^^^
SyntaxError: invalid syntax
[2024-12-18T15:45:48.869+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:45:48.890+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.034 seconds
[2024-12-18T15:46:44.936+0000] {processor.py:186} INFO - Started process (PID=3824) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:46:44.938+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:46:44.942+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:46:44.942+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:46:44.950+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:46:44.948+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 3
    from datetimeimport datetime
                        ^^^^^^^^
SyntaxError: invalid syntax
[2024-12-18T15:46:44.950+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:46:44.972+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.043 seconds
[2024-12-18T15:47:40.182+0000] {processor.py:186} INFO - Started process (PID=4287) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:47:40.184+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:47:40.186+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:47:40.186+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:47:40.190+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:47:40.189+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 3
    from datetimeimport datetime
                        ^^^^^^^^
SyntaxError: invalid syntax
[2024-12-18T15:47:40.190+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:47:40.213+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.037 seconds
[2024-12-18T15:48:33.421+0000] {processor.py:186} INFO - Started process (PID=4343) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:48:33.423+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:48:33.425+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:48:33.425+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:48:33.429+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:48:33.428+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 3
    from datetimeimport datetime
                        ^^^^^^^^
SyntaxError: invalid syntax
[2024-12-18T15:48:33.429+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:48:33.450+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.034 seconds
[2024-12-18T15:49:35.765+0000] {processor.py:186} INFO - Started process (PID=4399) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:49:35.766+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:49:35.769+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:49:35.768+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:49:35.772+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:49:35.771+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 3
    from datetimeimport datetime
                        ^^^^^^^^
SyntaxError: invalid syntax
[2024-12-18T15:49:35.772+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:49:35.793+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.034 seconds
[2024-12-18T15:50:07.928+0000] {processor.py:186} INFO - Started process (PID=4433) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:50:07.929+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:50:07.932+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:50:07.931+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:50:07.936+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:50:07.934+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 3
    from datetimeimport datetime
                        ^^^^^^^^
SyntaxError: invalid syntax
[2024-12-18T15:50:07.936+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:50:07.957+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.035 seconds
[2024-12-18T15:51:03.741+0000] {processor.py:186} INFO - Started process (PID=4502) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:51:03.743+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:51:03.745+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:51:03.744+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:51:03.749+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:51:03.747+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 9
    'start_date': datetime(2024,18,12),
                ^
SyntaxError: invalid syntax
[2024-12-18T15:51:03.749+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:51:03.769+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.034 seconds
[2024-12-18T15:51:29.508+0000] {processor.py:186} INFO - Started process (PID=4530) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:51:29.509+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:51:29.511+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:51:29.511+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:51:29.515+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:51:29.513+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 9
    'start_date': datetime(2024,18,12),
                ^
SyntaxError: invalid syntax
[2024-12-18T15:51:29.515+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:51:29.536+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.034 seconds
[2024-12-18T15:52:23.294+0000] {processor.py:186} INFO - Started process (PID=4586) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:52:23.295+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:52:23.297+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:52:23.297+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:52:23.301+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:52:23.300+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 9
    'start_date': datetime(2024,18,12),
                ^
SyntaxError: invalid syntax
[2024-12-18T15:52:23.302+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:52:23.326+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.039 seconds
[2024-12-18T15:53:19.075+0000] {processor.py:186} INFO - Started process (PID=4643) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:53:19.076+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:53:19.079+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:53:19.078+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:53:19.083+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:53:19.082+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 13, in <module>
    with DAG('simple_dag',
         ^^^^^^^^^^^^^^^^^
TypeError: DAG.__init__() got an unexpected keyword argument 'catch_up'
[2024-12-18T15:53:19.083+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:53:19.105+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.037 seconds
[2024-12-18T15:53:50.538+0000] {processor.py:186} INFO - Started process (PID=4680) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:53:50.539+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:53:50.541+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:53:50.541+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:53:50.546+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:53:50.544+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 13, in <module>
    with DAG('simple_dag',
         ^^^^^^^^^^^^^^^^^
TypeError: DAG.__init__() got an unexpected keyword argument 'catch_up'
[2024-12-18T15:53:50.546+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:53:50.570+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.038 seconds
[2024-12-18T15:54:22.213+0000] {processor.py:186} INFO - Started process (PID=4728) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:54:22.216+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:54:22.220+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:54:22.219+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:54:22.230+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:54:22.226+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 13, in <module>
    with DAG('simple_dag',
         ^^^^^^^^^^^^^^^^^
TypeError: DAG.__init__() got an unexpected keyword argument 'catch_up'
[2024-12-18T15:54:22.230+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:54:22.269+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.072 seconds
[2024-12-18T15:55:19.871+0000] {processor.py:186} INFO - Started process (PID=4795) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:55:19.873+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:55:19.875+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:55:19.874+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:55:19.878+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:55:19.877+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 13, in <module>
    with DAG('simple_dag',
         ^^^^^^^^^^^^^^^^^
TypeError: DAG.__init__() got an unexpected keyword argument 'catch_up'
[2024-12-18T15:55:19.879+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:55:19.899+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.034 seconds
[2024-12-18T15:56:13.584+0000] {processor.py:186} INFO - Started process (PID=4852) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:56:13.586+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:56:13.588+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:56:13.587+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:56:13.591+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:56:13.590+0000] {dagbag.py:387} ERROR - Failed to import: /home/ubuntu/airflow/dags/weather.py
Traceback (most recent call last):
  File "/home/ubuntu/demo8/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/airflow/dags/weather.py", line 13, in <module>
    with DAG('simple_dag',
         ^^^^^^^^^^^^^^^^^
TypeError: DAG.__init__() got an unexpected keyword argument 'catch_up'
[2024-12-18T15:56:13.592+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:56:13.613+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.035 seconds
[2024-12-18T15:56:49.638+0000] {processor.py:186} INFO - Started process (PID=4884) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:56:49.640+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:56:49.642+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:56:49.641+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:56:49.653+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:56:49.784+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:56:49.783+0000] {override.py:1911} INFO - Created Permission View: can read on DAG:simple_dag
[2024-12-18T15:56:49.797+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:56:49.797+0000] {override.py:1911} INFO - Created Permission View: can edit on DAG:simple_dag
[2024-12-18T15:56:49.807+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:56:49.807+0000] {override.py:1911} INFO - Created Permission View: can delete on DAG:simple_dag
[2024-12-18T15:56:49.824+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:56:49.823+0000] {override.py:1911} INFO - Created Permission View: can read on DAG Run:simple_dag
[2024-12-18T15:56:49.833+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:56:49.833+0000] {override.py:1911} INFO - Created Permission View: can create on DAG Run:simple_dag
[2024-12-18T15:56:49.843+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:56:49.843+0000] {override.py:1911} INFO - Created Permission View: can delete on DAG Run:simple_dag
[2024-12-18T15:56:49.854+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:56:49.853+0000] {override.py:1911} INFO - Created Permission View: menu access on DAG Run:simple_dag
[2024-12-18T15:56:49.854+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:56:49.854+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T15:56:49.867+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:56:49.866+0000] {dag.py:3262} INFO - Creating ORM DAG for simple_dag
[2024-12-18T15:56:49.876+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:56:49.875+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T15:56:49.901+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.269 seconds
[2024-12-18T15:57:47.389+0000] {processor.py:186} INFO - Started process (PID=4942) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:57:47.390+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:57:47.393+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:57:47.392+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:57:47.400+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:57:47.417+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:57:47.417+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T15:57:47.439+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:57:47.438+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T15:57:47.462+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.079 seconds
[2024-12-18T15:58:41.102+0000] {processor.py:186} INFO - Started process (PID=4997) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:58:41.103+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:58:41.105+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:58:41.105+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:58:41.113+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:58:41.129+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:58:41.129+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T15:58:41.150+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:58:41.150+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T15:58:41.171+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.075 seconds
[2024-12-18T15:59:48.535+0000] {processor.py:186} INFO - Started process (PID=5059) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:59:48.539+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T15:59:48.545+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:59:48.545+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:59:48.567+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T15:59:48.622+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:59:48.621+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T15:59:48.670+0000] {logging_mixin.py:190} INFO - [2024-12-18T15:59:48.668+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T15:59:48.710+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.188 seconds
[2024-12-18T16:00:46.208+0000] {processor.py:186} INFO - Started process (PID=5126) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T16:00:46.210+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T16:00:46.212+0000] {logging_mixin.py:190} INFO - [2024-12-18T16:00:46.211+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T16:00:46.219+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T16:00:46.235+0000] {logging_mixin.py:190} INFO - [2024-12-18T16:00:46.235+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T16:00:46.257+0000] {logging_mixin.py:190} INFO - [2024-12-18T16:00:46.256+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T16:00:46.279+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.077 seconds
[2024-12-18T16:01:39.911+0000] {processor.py:186} INFO - Started process (PID=5183) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T16:01:39.912+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T16:01:39.914+0000] {logging_mixin.py:190} INFO - [2024-12-18T16:01:39.914+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T16:01:39.922+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T16:01:39.938+0000] {logging_mixin.py:190} INFO - [2024-12-18T16:01:39.938+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T16:01:39.959+0000] {logging_mixin.py:190} INFO - [2024-12-18T16:01:39.959+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T16:01:39.982+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.077 seconds
[2024-12-18T16:02:33.127+0000] {processor.py:186} INFO - Started process (PID=5238) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T16:02:33.129+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T16:02:33.131+0000] {logging_mixin.py:190} INFO - [2024-12-18T16:02:33.130+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T16:02:33.138+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T16:02:33.160+0000] {logging_mixin.py:190} INFO - [2024-12-18T16:02:33.160+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T16:02:33.185+0000] {logging_mixin.py:190} INFO - [2024-12-18T16:02:33.185+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T16:02:33.207+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.086 seconds
[2024-12-18T16:03:28.467+0000] {processor.py:186} INFO - Started process (PID=5293) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T16:03:28.468+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T16:03:28.470+0000] {logging_mixin.py:190} INFO - [2024-12-18T16:03:28.470+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T16:03:28.478+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T16:03:28.494+0000] {logging_mixin.py:190} INFO - [2024-12-18T16:03:28.493+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T16:03:28.515+0000] {logging_mixin.py:190} INFO - [2024-12-18T16:03:28.514+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T16:03:28.536+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.075 seconds
[2024-12-18T20:29:15.418+0000] {processor.py:186} INFO - Started process (PID=2544) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:29:15.421+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:29:15.426+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:29:15.424+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:29:15.447+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:29:15.512+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:29:15.511+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:29:15.570+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:29:15.570+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:29:15.615+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.211 seconds
[2024-12-18T20:30:14.459+0000] {processor.py:186} INFO - Started process (PID=2782) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:30:14.461+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:30:14.463+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:30:14.463+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:30:14.471+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:30:14.487+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:30:14.486+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:30:14.507+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:30:14.507+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:30:14.532+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.079 seconds
[2024-12-18T20:31:09.888+0000] {processor.py:186} INFO - Started process (PID=3056) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:31:09.890+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:31:09.893+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:31:09.892+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:31:09.900+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:31:09.917+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:31:09.917+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:31:09.939+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:31:09.938+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:31:09.961+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.079 seconds
[2024-12-18T20:32:05.246+0000] {processor.py:186} INFO - Started process (PID=3250) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:32:05.248+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:32:05.250+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:32:05.250+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:32:05.258+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:32:05.274+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:32:05.273+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:32:05.294+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:32:05.294+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:32:05.316+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.075 seconds
[2024-12-18T20:33:00.529+0000] {processor.py:186} INFO - Started process (PID=3429) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:33:00.531+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:33:00.534+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:33:00.533+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:33:00.542+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:33:00.558+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:33:00.558+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:33:00.580+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:33:00.580+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:33:00.603+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.087 seconds
[2024-12-18T20:33:55.844+0000] {processor.py:186} INFO - Started process (PID=3546) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:33:55.846+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:33:55.848+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:33:55.848+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:33:55.856+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:33:55.872+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:33:55.872+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:33:56.005+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:33:56.005+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:33:56.027+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.188 seconds
[2024-12-18T20:34:30.864+0000] {processor.py:186} INFO - Started process (PID=3699) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:34:30.865+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:34:30.868+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:34:30.867+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:34:30.875+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:34:31.022+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:34:31.021+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:34:31.039+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:34:31.039+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:34:31.059+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.202 seconds
[2024-12-18T20:35:04.692+0000] {processor.py:186} INFO - Started process (PID=3736) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:35:04.693+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:35:04.695+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:35:04.695+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:35:04.703+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:35:04.720+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:35:04.720+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:35:04.750+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:35:04.750+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:35:04.773+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.088 seconds
[2024-12-18T20:36:02.536+0000] {processor.py:186} INFO - Started process (PID=3857) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:36:02.537+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:36:02.540+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:36:02.540+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:36:02.548+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:36:02.564+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:36:02.564+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:36:02.585+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:36:02.585+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:36:02.608+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.078 seconds
[2024-12-18T20:37:00.417+0000] {processor.py:186} INFO - Started process (PID=3976) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:37:00.419+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:37:00.422+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:37:00.421+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:37:00.429+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:37:00.446+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:37:00.445+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:37:00.467+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:37:00.467+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:37:00.489+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.078 seconds
[2024-12-18T20:37:53.057+0000] {processor.py:186} INFO - Started process (PID=4092) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:37:53.058+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:37:53.061+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:37:53.060+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:37:53.069+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:37:53.085+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:37:53.085+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:37:53.107+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:37:53.107+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:37:53.129+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.079 seconds
[2024-12-18T20:38:50.241+0000] {processor.py:186} INFO - Started process (PID=4211) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:38:50.242+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:38:50.245+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:38:50.244+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:38:50.253+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:38:50.269+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:38:50.269+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:38:50.291+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:38:50.291+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:38:50.312+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.078 seconds
[2024-12-18T20:39:45.799+0000] {processor.py:186} INFO - Started process (PID=4388) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:39:45.801+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:39:45.803+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:39:45.803+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:39:45.812+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:39:45.828+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:39:45.828+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:39:45.851+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:39:45.851+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:39:45.873+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.081 seconds
[2024-12-18T20:40:42.379+0000] {processor.py:186} INFO - Started process (PID=4524) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:40:42.381+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:40:42.383+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:40:42.383+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:40:42.391+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:40:42.409+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:40:42.408+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:40:42.431+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:40:42.431+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:40:42.454+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.081 seconds
[2024-12-18T20:41:36.545+0000] {processor.py:186} INFO - Started process (PID=4644) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:41:36.547+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:41:36.549+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:41:36.549+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:41:36.557+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:41:36.574+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:41:36.573+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:41:36.604+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:41:36.604+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:41:36.630+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.092 seconds
[2024-12-18T20:42:29.617+0000] {processor.py:186} INFO - Started process (PID=4757) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:42:29.619+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:42:29.621+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:42:29.621+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:42:29.629+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:42:29.645+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:42:29.645+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:42:29.667+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:42:29.667+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:42:29.688+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.077 seconds
[2024-12-18T20:43:23.575+0000] {processor.py:186} INFO - Started process (PID=4874) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:43:23.577+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:43:23.580+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:43:23.579+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:43:23.587+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:43:23.604+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:43:23.604+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:43:23.626+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:43:23.626+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:43:23.650+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.081 seconds
[2024-12-18T20:44:17.370+0000] {processor.py:186} INFO - Started process (PID=4953) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:44:17.371+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:44:17.374+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:44:17.373+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:44:17.384+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:44:17.427+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:44:17.425+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:44:17.469+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:44:17.469+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:44:17.492+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.128 seconds
[2024-12-18T20:45:11.550+0000] {processor.py:186} INFO - Started process (PID=5112) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:45:11.552+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:45:11.558+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:45:11.556+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:45:11.571+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:45:11.588+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:45:11.588+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:45:11.616+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:45:11.615+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:45:11.645+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.101 seconds
[2024-12-18T20:46:10.941+0000] {processor.py:186} INFO - Started process (PID=5356) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:46:10.944+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:46:10.949+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:46:10.947+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:46:10.969+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:46:11.027+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:46:11.027+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:46:11.068+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:46:11.068+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:46:11.112+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.182 seconds
[2024-12-18T20:46:51.809+0000] {processor.py:186} INFO - Started process (PID=5568) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:46:51.813+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:46:51.818+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:46:51.818+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:46:51.840+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:46:51.890+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:46:51.890+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:46:51.938+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:46:51.938+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:46:51.982+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.196 seconds
[2024-12-18T20:47:52.072+0000] {processor.py:186} INFO - Started process (PID=5861) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:47:52.074+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:47:52.076+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:47:52.076+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:47:52.084+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:47:52.101+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:47:52.101+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:47:52.124+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:47:52.123+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to 2024-12-18 00:00:00+00:00, run_after=2024-12-18 00:00:00+00:00
[2024-12-18T20:47:52.148+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.081 seconds
[2024-12-18T20:48:50.616+0000] {processor.py:186} INFO - Started process (PID=6308) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:48:50.617+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:48:50.620+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:48:50.619+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:48:50.628+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:48:50.646+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:48:50.646+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:48:50.668+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:48:50.668+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to None, run_after=None
[2024-12-18T20:48:50.693+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.084 seconds
[2024-12-18T20:49:43.431+0000] {processor.py:186} INFO - Started process (PID=6457) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:49:43.432+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:49:43.435+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:49:43.435+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:49:43.444+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:49:43.460+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:49:43.460+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:49:43.483+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:49:43.483+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to None, run_after=None
[2024-12-18T20:49:43.507+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.082 seconds
[2024-12-18T20:50:41.611+0000] {processor.py:186} INFO - Started process (PID=6821) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:50:41.613+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:50:41.615+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:50:41.615+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:50:41.623+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:50:41.640+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:50:41.640+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:50:41.662+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:50:41.662+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to None, run_after=None
[2024-12-18T20:50:41.686+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.080 seconds
[2024-12-18T20:51:33.221+0000] {processor.py:186} INFO - Started process (PID=7037) to work on /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:51:33.223+0000] {processor.py:914} INFO - Processing file /home/ubuntu/airflow/dags/weather.py for tasks to queue
[2024-12-18T20:51:33.225+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:51:33.225+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:51:33.233+0000] {processor.py:925} INFO - DAG(s) 'simple_dag' retrieved from /home/ubuntu/airflow/dags/weather.py
[2024-12-18T20:51:33.250+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:51:33.250+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-18T20:51:33.272+0000] {logging_mixin.py:190} INFO - [2024-12-18T20:51:33.272+0000] {dag.py:4180} INFO - Setting next_dagrun for simple_dag to None, run_after=None
[2024-12-18T20:51:33.295+0000] {processor.py:208} INFO - Processing /home/ubuntu/airflow/dags/weather.py took 0.079 seconds
